index.html

```diff
@@ -8,8 +8,10 @@
   <meta name="keywords" content="canvas,confetti,component,module,animation,javascript,catdad">
 
   <meta name="theme-color" content="#393939">
-
-  <title>🎊</title>
+   
+  <link rel="icon" href="logo.png" type="image/gif" sizes="16x16">
```

Comment:
I am confused. The extension is a png but the type is set to gif? Also, the actual file that was uploaded shows as a binary file rather than an image, which doesn't seem correct.

Reply:
Sorry, let me check.